docs/en/api/S3-API.md

```diff
@@ -18,375 +18,1701 @@ sharing with file system based applications, and storage system abstraction (e.g
+
+## Global request headers
+<table class="table table-striped">
+  <tr>
+    <th>Header</th>
+    <th>Content</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td><a href="https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-auth-using-authorization-header.html">Authorization</a></td>
+    <td>AWS4-HMAC-SHA256
+    Credential=<b>{user}</b>/...,
+    SignedHeaders=...,
+    Signature=...</td>
+    <td>There is currently no support for access & secret keys in the Alluxio S3 API.
+    The only supported authentication scheme is the <a href="{{ '/en/operation/Security.html#simple' | relativize_url }}">SIMPLE</a>
+    authentication type. By default, the user that is used to perform any operations is the user that was used to
+    launch the Alluxio proxy process.
+    <br/><br/>
+    Therefore this header is used exclusively to specify an Alluxio ACL username to perform an
+    operation with. In order to remain compatible with other S3 clients, the header is still
+    expected to follow the
+    <a href="https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-auth-using-authorization-header.html">AWS Signature Version 4</a>
+    format.
+    <br/><br/>
+    When supplying an access key to an S3 client, put the intended Alluxio ACL username.
+    The secret key is unused so you may use any dummy value.</td>
+  </tr>
+</table>
+
+## Supported S3 API Actions
+
+The following table describes the support status for current [S3 API Actions](https://docs.aws.amazon.com/AmazonS3/latest/API/API_Operations.html):
 
 <table class="table table-striped">
-  <tr><th>S3 Feature</th><th>Status</th></tr>
-{% for item in site.data.table.s3-api-supported-operations %}
   <tr>
-    <td>{{ item.S3Feature }}</td>
-    <td>{{ item.Status }}</td>
+    <th>S3 API Action</th>
+    <th>REST API Endpoint</th>
+    <th>Supported Headers</th>
+    <th>Supported Query Parameters</th>
+  </tr>
+{% for item in site.data.table.s3-api-supported-actions %}
+  <tr>
+    <td><a href="https://docs.aws.amazon.com/AmazonS3/latest/API/API_{{ item.action }}.html">{{ item.action }}</a></td>
+    <td>{{ item.endpoint }}</td>
+    <td>
+      {% assign headers = item.headers | split: "|" %}
+      {% if headers.size == 0 %}
+      N/A
+      {% else %}
+      <ul style="list-style-type:none;margin:0;padding:0">
+      {% for header in headers %}
+      {% if forloop.last %}
+      <li>{{ header }}</li>
+      {% else %}
+      <li>{{ header }},</li>
+      {% endif %}
+      {% endfor %}
+      </ul>
+      {% endif %}
+    </td>
+    <td>
+      {% assign params = item.queryParams | split: "|" %}
+      {% if params.size == 0 %}
+      N/A
+      {% else %}
+      <ul style="list-style-type:none;margin:0;padding:0">
+      {% for param in params %}
+      {% if forloop.last %}
+      <li>{{ param }}</li>
+      {% else %}
+      <li>{{ param }},</li>
+      {% endif %}
+      {% endfor %}
+      </ul>
+      {% endif %}
+    </td>
```

Comment:
The for loops in these two `<td>` blocks are just trying to process the `|`-separated entries of the csv.

The if-else blocks are just handling whether or not to append a trailing `,`.